contracts/eth/nearbridge/test/NearBridge2.js

```diff
@@ -87,68 +84,64 @@ it('Add second block in first epoch should be verifiable', async function () {
 });
 
 it('Test adding blocks in new epoch when bps change', async function () {
-    // Skip until tests are upgraded having blocks after nearcore 1.23.0
-    this.skip();
 
-    const block181 = require('./181.json');
-    const block244 = require('./244.json');
-    const block304 = require('./304.json');
-    const block308 = require('./308.json');
-    const block368 = require('./368.json');
-    const block369 = require('./369.json');
+    const block126313429 = require('./126313429.json');
+    const block126315547 = require('./126315547.json');
```

Comment:
Should we update the block244 -> block126315547, block304 -> block126315744, block308 -> block126315811 in all tests? Or there is a reason to keep this files different?

Reply:
The test is about adding blocks from the first epoch, so I think we can keep it for this